docs/DeveloperGuide.md

```diff
@@ -574,6 +570,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * Steps 1a1-1a2 are repeated until the increment amount entered is valid.
     * Use case resumes from step 2.
 
+**Use case: UC6 - View Command History**
+
+**MSS**
+
+1. User requests to view the command history of a certain number of past commands.
+1. EmployeeManager validates the input and fetches the required number of recent valid commands from the CommandHistory.
```

Comment:
I think it is a bit confusing that "EmployeeManager validates the input" but there is an extension 1a which implies that the validity is checked in step 1. Would it be better to remove "EmployeeManager validates the input"?

Reply:
Okay that makes sense!